packages/eslint-config-airbnb/rules/best-practices.js

```diff
@@ -31,8 +37,9 @@ module.exports = {
     'no-div-regex': 0,
     // disallow else after a return in an if
     'no-else-return': 2,
-    // disallow use of labels for anything other then loops and switches
-    'no-empty-label': 2,
+    // disallow Unnecessary Labels
+    // http://eslint.org/docs/rules/no-extra-label
+    'no-extra-label': 0,
```

Comment:
since we already prevent labels in `no-labels`, i think this can be set to `2` without any additional justification in the guide (assuming it already says "no labels ever" somewhere)